kanidm_client/src/lib.rs

```diff
@@ -701,6 +705,26 @@ impl KanidmClient {
         )
     }
 
+    pub fn idm_account_primary_credential_generate_backup_code(
+        &self,
+        id: &str,
+    ) -> Result<Vec<String>, ClientError> {
+        tokio_block_on(
+            self.asclient
+                .idm_account_primary_credential_generate_backup_code(id),
```

Comment:
no need for the trailing ',' here

Reply:
I cannot remove it - my editor puts it back :laughing: